docs/installation_and_admin_guide.md

```diff
@@ -149,6 +149,10 @@ Once downloaded the source code, you can install WireCloud using the `setup.py`
 
 Where `<version>` is the version of WireCloud to install.
 
+### Installing WireCloud with Docker
+
+WireCloud can be deployed using [Docker](https://www.docker.com/), the images can be found in the [wirecloud hub](https://hub.docker.com/r/wirecloud/fiware-wirecloud/), the DockerFile's are in the [docker-wirecloud](https://github.com/Wirecloud/docker-wirecloud/) repository, and the documentation to deploy and start it are in this [README.md](https://github.com/Wirecloud/docker-wirecloud/tree/master/hub-docs/README.md)
```

Comment:
- Installing WireCloud with Docker => Installing WireCloud using Docker (to follow the same pattern as other sections, e.g. Installing WireCloud using pip)
- what is wirecloud hub? do you mean docker hub?, "the images can be found **at** docker hub.."
- plural of DockerFile is DockerFiles o docker files, "the DockerFiles are hosted in the docker-wirecloud repository"
- "the documentation.... is in this README.md file"